app/config/collections.php

```diff
@@ -2246,132 +2590,361 @@
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('search'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 16384,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('version'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 8,
+                'signed' => true,
+                'required' => false,
+                'default' => 'v3',
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                'array' => false,
+                '$id' => ID::custom('entrypoint'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 16384,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'filters' => [],
+            ],
+            [
+                'array' => false,
+                '$id' => ID::custom('buildCommand'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 16384,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'filters' => [],
+            ],
+            [
+                'array' => false,
+                '$id' => ID::custom('installCommand'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 16384,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'filters' => [],
+            ],
+        ],
+        'indexes' => [
+            [
+                '$id' => ID::custom('_key_search'),
+                'type' => Database::INDEX_FULLTEXT,
+                'attributes' => ['search'],
+                'lengths' => [],
+                'orders' => [Database::ORDER_ASC],
+            ],
+            [
+                '$id' => ID::custom('_key_name'),
+                'type' => Database::INDEX_KEY,
+                'attributes' => ['name'],
+                'lengths' => [768],
+                'orders' => [Database::ORDER_ASC],
+            ],
+            [
+                '$id' => ID::custom('_key_enabled'),
+                'type' => Database::INDEX_KEY,
+                'attributes' => ['enabled'],
+                'lengths' => [],
+                'orders' => [Database::ORDER_ASC],
+            ],
+            [
+                '$id' => ID::custom('_key_vcsInstallationId'),
+                'type' => Database::INDEX_KEY,
+                'attributes' => ['vcsInstallationId'],
+                'lengths' => [Database::LENGTH_KEY],
```

Comment:
Since '_key_vcsInstallationId' size is 255 = Database::LENGTH_KEY
We do not need to specify an index length , since it will be the same as the attribute length by default, and while we are less than 768 , we are ok.